webserver/webserver/src/main/java/io/helidon/webserver/SocketConfiguration.java

```diff
@@ -118,6 +120,13 @@ default String name() {
      */
     Set<String> enabledSslProtocols();
 
+    /**
+     * Whether to require client authentication or not.
+     *
+     * @return client authentication
+     */
+    ClientAuth clientAuth();
```

Comment:
Netty APIs should not be part of Helidon APIs

Reply:
updated